src/core/ui/content/PageContentBlockHeader.tsx

```
@@ -1,59 +1,67 @@
 
-export interface PageContentBlockHeaderProps {
-  title: ReactNode;
-  icon?: ReactElement<SvgIconProps>;
-  actions?: ReactNode;
-  disclaimer?: ReactNode;
-  fullWidth?: boolean;
-}
+import { gutters } from '../grid/utils';
+import { useNextBlockAnchor } from '../keyboardNavigation/NextBlockAnchor';
 
 const PageContentBlockHeader = <D extends React.ElementType = BoxTypeMap['defaultComponent'], P = {}>({
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)